### PR TITLE
SPOI-10700 changing used abstract FSLoader to JsonFSLoader

### PR DESCRIPTION
**Pull request by oliverwnk**
@yogidevendra please review and merge as appropriate

### CODE REVIEW

File: tutorials/enricher/pom.xml [hunk 1]
```diff
@@ -13,9 +13,9 @@
 
   <properties>
     <!-- change this if you desire to use a different version of Apex Core -->
-    <apex.version>3.4.0</apex.version>
+    <apex.version>3.6.0-dt20161215</apex.version>
```
Comment (amberarrow) [inline tutorials/enricher/pom.xml]: We shouldn't be using special build versions; if the latest release (e.g. 3.5.0) is not adequate we should use 3.6.0-SNAPSHOT

Comment (sanjaypujare) [inline tutorials/enricher/pom.xml]: @amberarrow  this is for SPOI-10583. The goal was to validate the examples with the latest apex-core, apex-malhar versions released by DT in RTS and not the open-source ones.

Comment (amberarrow) [inline tutorials/enricher/pom.xml]: Using that version for testing and validation is OK but these examples are supposed to be usable by anyone, not just RTS users. After testing, what gets committed should be a publicly accessible version. If someone tries to build with that apex version they get an error:

[ERROR] Failed to execute goal on project enricher: Could not resolve dependencies for project com.example:enricher:jar:1.0-SNAPSHOT: The following artifacts could not be resolved: org.apache.apex:apex-common:jar:3.6.0-dt20161215, org.apache.apex:apex-engine:jar:3.6.0-dt20161215: Could not find artifact org.apache.apex:apex-common:jar:3.6.0-dt20161215 in central (https://repo.maven.apache.org/maven2) -> [Help 1]

Comment (sanjaypujare) [inline tutorials/enricher/pom.xml]: @amberarrow  Hmmm, will need to discuss this with @priteshm and others then. BTW, is it possible to get the dt versions by adding another maven repository?

**Pull request closed without merging**